### ISSUE TITLE
about loss implementation

### ISSUE THREAD

**Issue by crackhopper** [crackhopper/TFS-toolbox#32]
what's the argument `with_logits` for?

    def __init__(self,netobj,with_logits=True):
        Loss.__init__(self,netobj,with_logits)

you don't use the parameter.

@mingzailao
